### PR TITLE
Ingest headers must always get the cached and refreshed token

### PR DESCRIPTION
**Pull request by aaclan-ebi**
Retrieving the headers every request should be fine since token manager inside hca-ingest is handling the caching and refreshing the token (every ~1hr)

https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/dcp/-/jobs/19037

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -25,6 +25,7 @@ dcp_wide_test_SS2:
     - integration
     - staging
     - prod
+    - aa-token-refresh
```
Comment (parthshahva) [inline .gitlab-ci.yml]: remove this before merging

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -34,6 +35,7 @@ dcp_wide_test_10x:
...
```
Comment (parthshahva) [inline .gitlab-ci.yml]: remove this too 🙂

Comment (aaclan-ebi) [reply .gitlab-ci.yml]: Yes! Thank you! :)

**Comment by aaclan-ebi** [comment]
Successful job ( it has been waiting for the secondary bundle but didn't fail, fix worked!)
https://allspark.dev.data.humancellatlas.org/HumanCellAtlas/dcp/-/jobs/19036

**Pull request merged**